wremnants/datasets/datasetDict_v9.py

```diff
@@ -101,3 +102,38 @@
                          'group' : "Diboson",
     }
 }
+
```

Comment:
Can't we get away with only changing the base bath rather than having the full path be different? This would be better for maintainability